library/core/src/slice/mod.rs

```diff
@@ -2069,14 +2069,14 @@ impl<T> [T] {
     ///         v == [-3, -5, 1, 4, 2] ||
     ///         v == [-5, -3, 1, 4, 2]);
     /// ```
-    #[unstable(feature = "slice_partition_at_index", issue = "55300")]
+    #[unstable(feature = "slice_select_nth_unstable", issue = "55300")]
```

Comment:
Since #55300 has already been FCP'ed to stabilize, it's better just mark all methods as `#[stable]`, wait until the FCP ends, and merge this as the stabilization PR.